fetch_data/main.py

```diff
@@ -0,0 +1,38 @@
+import base64
+import requests
+from google.cloud import storage
+from dotenv import load_dotenv
+load_dotenv()
+import os
+
+url = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv'
+gcp_project = "test-python-etl"
+service_account = "service-account_local-dev"
+
+def fetch_data(input):
+    data = requests.get(input)
+    text = data.text
+    return text
+
+def store_data(data):
+    try:
+        # Instantiates a client
+        storage_client = storage.Client()
+        bucket = storage_client.get_bucket("bucket-data_raw")
+        bucket_csv = bucket.blob('data_us_nyt.csv')
+        bucket_csv.upload_from_string(data)
+        print("complete!")
+    except Exception as e:
+        print(str(e))
+
+def hello_pubsub(event, context):
```

Comment:
Can you delete this function?

Reply:
oh ok

Reply:
if you insist!

Reply:
I hated that function from the very beginning!